examples/telegraf/zookeeper/zookeeper_data_size_batch.tick

```diff
@@ -0,0 +1,47 @@
+// zookeeper_data_size_batch
+
+// metric: "approximate_data_size"
+// available_fields: "avg_latency","ephemerals_count","max_file_descriptor_count","max_latency","min_latency","num_alive_connections","open_file_descriptor_count","outstanding_requests","packets_received","packets_sent","version","watch_count","znode_count" 
+
+// TELEGRAF CONFIGURATION
+// [inputs.zookeeper]
+//   servers = [":2181"]
+
+// DEFINE: kapacitor define zookeeper_data_size_batch -type batch -tick zookeeper/zookeeper_data_size_batch.tick -dbrp telegraf.autogen
+// ENABLE: kapacitor enable zookeeper_data_size_batch
+
+// Parameters
+var info = 100000000 // 100MB
+var warn = 1000000000 // 1GB 
+var crit = 10000000000 // 10GB
+var infoSig = 2.5
+var warnSig = 3
+var critSig = 3.5
+var period = 10s
+var every = 10s
+
+// Dataframe
+var data = batch
+  |query('''SELECT mean(approximate_data_size) AS stat FROM "telegraf"."autogen"."zookeeper"''')
+    .period(period)
+    .every(every)
+    .groupBy('host')
+        
+// Thresholds
+var alert = data
+  |eval(lambda: sigma("stat"))
```

Comment:
An alternative approach would be to dived the stat here by `mb` or `gb` and then set the thresholds in those units.

```
var info = 0.1 // 100 MB
var warn = 1 // 1 GB
var crit = 10 // 1 GB
```

```
|eval(lambda:"stat" / gb, lambda: sigma("stat"))
   .as('stat', 'sigma')
|...
```